Apps/W1/HybridGP/app/src/tables/GPCompanyAdditionalSettings.table.al

```diff
@@ -283,6 +291,35 @@ table 40105 "GP Company Additional Settings"
                     Rec.Validate("Migrate Inventory Module", true);
             end;
         }
+        field(29; "Oldest Hist. Year to Migrate"; Integer)
```

Comment:
seems confusing..."Oldest Hist. Year to Migrate" and "Oldest GL Year to Migrate". Are these to be associated with OPEN vs CLOSED years in GP or is it more than that?

Reply:
if 'Hist' needs to be in the field it could be either one. I am fine with leaving it as is. Just reconciling these two options - tell me the oldest year to migrate; and then tell me the oldest history year to migrate. I just thought we should be using OPEN and CLOSED instead of 'hist'. I would leave it as is.